02_calculator/calculator.rb

```diff
@@ -1 +1,27 @@
-#write your code here
+def add(a, b)
+    a + b
+end
+
+def subtract(a, b)
+    a - b
+end
+
+def sum(arr)  
+    if(arr.length <= 0)
+        return 0
+    end
+    return arr.inject { |memo, n| memo + n }      
```

Comment:
If you passed in an argument like in `multiply` below, then you wouldn't need to handle the length == 0 case. Also, in Ruby you don't need the `return` keyword if it's the last expression.

```suggestion
    arr.inject(0) { |memo, n| memo + n }      
```